catroid/build.gradle

```diff
@@ -86,7 +86,7 @@ android {
             buildConfigField "boolean", "FEATURE_RASPI_ENABLED", "true"
             buildConfigField "boolean", "FEATURE_POCKETMUSIC_ENABLED", "true"
             buildConfigField "boolean", "FEATURE_CAST_ENABLED", "true"
-            buildConfigField "boolean", "CRASHLYTICS_CRASH_REPORT_ENABLED", "false"
+            buildConfigField "boolean", "CRASHLYTICS_CRASH_REPORT_ENABLED", "true"
```

Comment:
Please create a separate pull request for this.
Also explain why this change is necessary, what the problem was and the effects of the patch. Also mention that the Jenkins Espresso testrun is green because the auto-generated AllEspressoTestsSuite.java file isn't up-to-date and doesn't contain the SettingsActivityTest. See [information in commit messages](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines#information-in-commit-messages).